### ISSUE TITLE
feat: has access right / error codes for all errors

### ISSUE THREAD

**Issue by stephan-fischer** [capacitor-community/media#86]
**Is your feature request related to a problem? Please describe.**
If the user dont allow the access to the gallery the `savePhoto` function is only returning a 
error with this hardcoded message: _Access to photos not allowed by user_

**Describe the solution you'd like**
To have a function, too check  the access like:

```ts
const access = await Media.requestAccess();
if (access) {
// save photo...
}
```

Or another solution is to have instead of the error message an error code for each
error like:
```objectivec
call.reject("Access to photos not allowed by user", "accessDenied")
```
And with this code the returned error could be checked.
Currently the check could only via string like:
```ts
   if(error.message === 'Access to photos not allowed by user') {
                // show dialog...
   }
```

**Comment by stephan-fischer** [comment]
Or could i create a pull request with error codes like:
call.reject("Access to photos not allowed by user", "accessDenied")

**Comment by nkalupahana** [comment]
@stephan-fischer that sounds great! If you have time to do that this week, go for it -- otherwise, I should have time this week to do it. Just let me know.

**Comment by nkalupahana** [comment]
Alright, I'll take this on this today :)

**Comment by nkalupahana** [comment]
Error codes added and published under 6.0.0.
